CalibCalorimetry/EcalPedestalOffsets/src/testChannel.cc

```diff
@@ -66,8 +68,7 @@ void testChannel::analyze(edm::Event const &event, edm::EventSetup const &eventS
 
   // get the headers
   // (one header for each supermodule)
-  edm::Handle<EcalRawDataCollection> DCCHeaders;
-  event.getByLabel(m_headerProducer, DCCHeaders);
+  const edm::Handle<EcalRawDataCollection> &DCCHeaders = event.getHandle(m_headerProducerToken);
   if (!DCCHeaders.isValid()) {
     edm::LogError("testChannel") << "Error! can't get the product " << m_headerProducer.c_str();
```

Comment:
One could avoid propagating the string as class member by wrting something like:
```suggestion
    edm::LogError("testChannel") << "Error! can't get the product for DCCHeaders";
```